src/main/java/fi/aalto/cs/apluscourses/presentation/ReplConfigurationFormModel.java

```diff
@@ -0,0 +1,115 @@
+package fi.aalto.cs.apluscourses.presentation;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.module.ModuleManager;
+import com.intellij.openapi.project.Project;
+import fi.aalto.cs.apluscourses.ui.repl.ReplConfigurationForm;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.jetbrains.annotations.NotNull;
+
+public class ReplConfigurationFormModel {
+
+  private Project project;
+  private String moduleWorkingDirectory;
+  private String targetModuleName;
+  private List<String> moduleNames;
+  public boolean startRepl = true;
```

Comment:
Nit: this should probably be private since there is already a `setStartRepl` method